crest/Assets/Crest/Crest/Scripts/Helpers/PropertyWrapper.cs

```diff
@@ -21,11 +21,34 @@ public interface IPropertyWrapper
         void SetInt(int param, int value);
     }
 
+    static class PropertyWrapperConstants
+    {
+        internal const string NO_SHADER_MESSAGE = "Cannot create required material because shader {0} could not be found or loaded."
+            + " Try right clicking the Crest folder in the Project view and selecting Reimport, and checking for errors.";
+    }
+
     [System.Serializable]
     public class PropertyWrapperMaterial : IPropertyWrapper
     {
         public PropertyWrapperMaterial(Material target) { material = target; }
-        public PropertyWrapperMaterial(Shader shader) { material = new Material(shader); }
+        public PropertyWrapperMaterial(Shader shader)
+        {
+            Debug.Assert(shader != null, "Cannot create required material because shader is null");
```

Comment:
preference for "material property wrapper" instead of just "material" as one might confuse this for a general unity message (maybe) if it got reported on discord

Reply:
It is now prefixed with "Crest.PropertyWrapperMaterial: " similar to other debug/asserts we have. Will that work?

Reply:
sure, its not showing me that change on github but im probably just failing at it. but what you described sounds good. one of these days ill do a pass over all asserts/logs to add a prefix!

Reply:
Thanks. Same. I had to go to the files tab otherwise it will show the changes at time of review.